NitroxClient/Communication/Packets/Processors/SpawnConsoleCommandProcessor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class SpawnConsoleCommandProcessor : ClientPacketProcessor<SpawnConsoleCommandEvent>
+    {
+        private readonly IPacketSender packetSender;
+
+        public SpawnConsoleCommandProcessor(IPacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public override void Process(SpawnConsoleCommandEvent packet)
+        {
+            GameObject gameObject = SerializationHelper.GetGameObject(packet.SerializeData);
+            LargeWorldEntity.Register(gameObject);
+            CrafterLogic.NotifyCraftEnd(gameObject, CraftData.GetTechType(gameObject));
+            gameObject.SendMessage("StartConstruction", SendMessageOptions.DontRequireReceiver);
```

Comment:
This will not add the vehicle to the VehicleModles list in Vehicles class if the serialzed object is an vehicle (including cyclops). That could cause syncing errors